<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComparePage</class>
 <widget class="QWidget" name="ComparePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>716</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ComparePage</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget

QWidget#compareListPanel
{
	border: 1px solid #dadada;
}

QWidget#topBar,#duelTopBar
{
	background: #2a5674;
}

QWidget#searchPanel
{
	border-width: 1px 1px 0px 0px;
	border-style: solid;
	border-color: #dadada;
}

QWidget#deviceHeaderWidget,#yourDeviceLabel,#otherDeviceLabel
{
	background-color: #d4eaf9;
	font-size:14px;
}

QLabel#sectionTitleLabel,#duelSectionTitleLabel
{
	background: #2a5674;
	color: #fff;
	font-weight: bold;
	font-size: 16px;
}

QPushButton#duelSectionCloseButton,#duelSectionHiddenButton {
	background-color: #2a5674;
	border: none;
	min-width: 70px; max-width: 70px;
	color: #fff;
	font-weight: normal;
	font-size: 16px;
}

QPushButton#duelSectionCloseButton:hover
{
	background-color: #2d5c7c;
}

QPushButton#duelSectionCloseButton:pressed
{
	background-color: #27516d;
}

QPushButton#duelSectionHiddenButton
{
	color: #2a5674;
}

QLabel#bottomLine1
{
	background: #4389b9;
}

QLabel#bottomLine2
{
	background: #54abe7;
}

QLabel#verticalLine
{
	background: #ccc;
}

QLabel#header
{
	background: #fff;
	color: #67b9da;
	font-family: &quot;Segoe UI&quot;;
	font-size: 24px;
	padding-top: 12px;
}

QLineEdit#searchLineEdit
{
	font-size: 14px;
	border: 1px solid #ccc;
	background-color: #fff;
	outline: none;
}

QLabel#loupeLabel
{
	border-image: url(:/loupe.png) 0 0 0 0 stretch stretch;
}

QWidget#testListWidget,#compareScrollWidget,#duelScrollWidget,#searchPanel
{   
    background: #ffffff;
}

QWidget#scrollAreaWidgetContets,#widget
{
	background: #ffffff;
}

QScrollBar:vertical
{
	border-radius: 3px;
	background: #eee;
	width: 27px;
	margin-left: 10px;
	margin-right: 10px;
	margin-top: 10px;
	margin-bottom: 10px;
}

QScrollBar::handle:vertical
{
	border-radius: 3px;
	background: #ccc;
	min-height: 340px;
}

QScrollBar::add-line:vertical
{
	border: none;
	height: 0px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical
{
	border: none;
	height: 0px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical
{
	border: none;
	width: 0px;
	height: 0px;
	background: none;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
{
	border: none;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="comparePanel" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="topBar" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>44</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>44</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="sectionTitleLabel">
              <property name="text">
               <string>Compare</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="SelectableScrollWidget" name="testListWidget" native="true"/>
           </item>
           <item>
            <widget class="QWidget" name="compareListPanel" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>1</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="ScrollWidget" name="compareScrollWidget" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="searchPanel" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>10</number>
                 </property>
                 <property name="topMargin">
                  <number>10</number>
                 </property>
                 <property name="rightMargin">
                  <number>1</number>
                 </property>
                 <property name="bottomMargin">
                  <number>10</number>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="searchLineEdit">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>32</height>
                    </size>
                   </property>
                   <property name="placeholderText">
                    <string>Type here to filter result</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="loupeLabel">
                   <property name="minimumSize">
                    <size>
                     <width>32</width>
                     <height>32</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="duelPanel" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="duelTopBar" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>44</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>44</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="duelSectionHiddenButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Close</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="duelSectionTitleLabel">
              <property name="text">
               <string>Duel</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="duelSectionCloseButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Close</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="deviceHeaderWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>54</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>54</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>6</number>
            </property>
            <property name="topMargin">
             <number>6</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="yourDeviceLabel">
              <property name="text">
               <string>Your device</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="otherDeviceLabel">
              <property name="text">
               <string>Other device</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="ScrollWidget" name="duelScrollWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="bottomLine1">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="bottomLine2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>2</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>2</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SelectableScrollWidget</class>
   <extends>QWidget</extends>
   <header location="global">SelectableScrollWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScrollWidget</class>
   <extends>QWidget</extends>
   <header location="global">ScrollWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
